docs/guides/1227-empty-buildkit-cache.md

```diff
@@ -0,0 +1,22 @@
+---
+slug: /1227/empty-buildkit-cache
+displayed_sidebar: europa
```

Comment:
This should be `displayed_sidebar: '0.2'`

Reply:
`'displayed_sidebar: 0.2' returns me an error